### ISSUE TITLE
Cut image-spec release with reference types 

### ISSUE THREAD

**Issue by lachie83** [opencontainers/image-spec#952]
Now that the reference types working group proposal has been merged into main, I would like to start the discussion on getting image-spec and corresponding [distribution-spec](https://github.com/opencontainers/distribution-spec/issues/347) releases cut. The reference types work was merged via the following PRs:

* https://github.com/opencontainers/image-spec/issues/940
* https://github.com/opencontainers/distribution-spec/pull/335

There is also some cleanup work covered in the following issue - https://github.com/opencontainers/image-spec/issues/940

In order to ease the decision of when to cut a release, I thought creating a checklist of items that must happen prior to release (release gating criteria if you will) might be a good way to make progress towards that goal. I have come up with the following list and am happy to add more items that might have been missed.

- [ ] Closure of outstanding tasks
- [ ] Conformance tests
- [ ] Sufficient Documentation
- [ ] OPTIONAL Implementation endorsements (potential list of registry providers that have agreed to implement the changes)
- [ ] Versioning agreement (if/how these changes to the image-spec constitute an API version change)

**Comment by sajayantony** [comment]
As a part of today's call  this issue was brought up and corresponding voting items were opened for cutting an RC1. Once we have implementors and feedback/endorsements we can bring up the discussion for a `v1.1.0` for e.g. have projects like K8s consume it was what @mikebrow suggested :) 
 - https://github.com/opencontainers/image-spec/issues/954 was opened as a follow up to this discussion. But I would keep this issue open until the vote/tag is done :)

**Comment by lachie83** [comment]
Associated rc1 PR https://github.com/opencontainers/image-spec/pull/953

**Comment by sajayantony** [comment]
Can we close this now that we have - https://github.com/opencontainers/image-spec/tree/v1.1.0-rc1
and - https://github.com/opencontainers/distribution-spec/tree/v1.1.0-rc1

**Comment by sajayantony** [comment]
@lachie83  - Can link https://github.com/opencontainers/distribution-spec/issues/139 as the conformance test item.

**Comment by jdolitsky** [comment]
this is meta can we close/remove?

**Comment by sajayantony** [comment]
Last pending PR before closing this #710

**Comment by sajayantony** [comment]
Closed with #1161
